addons/medical/scripts/Game/ACE_Medical/Components/Gadgets/ACE_Medical_ConsumableEpinephrine.c

```diff
@@ -32,12 +32,17 @@ class ACE_Medical_ConsumableEpinephrine : SCR_ConsumableEffectHealthItems
 			failReason = SCR_EConsumableFailReason.IS_BLEEDING;
 			return false;
 		}
-		
-		// Cannot be applied if critically injured
-		// We also have to check if the character is still healable
-		if (damageManager.ACE_Medical_HasCriticalHealth() && damageManager.ACE_Medical_CanBeHealed())
+
+		// In the event the patient is critically injured, we don't allow epi
+		if (damageManager.ACE_Medical_HasCriticalHealth() && damageManager.ACE_Medical_CanBeHealed(0.999))
 		{
-			failReason = SCR_EConsumableFailReason.ACE_MEDICAL_TOO_DAMAGED;
+			// If the patient can still be healed with the medicalkit, we'll just say that the patient is too injured
+			if (damageManager.ACE_Medical_CanBeHealed())
+				failReason = SCR_EConsumableFailReason.ACE_MEDICAL_TOO_DAMAGED;
+			// Otherwise, we'll display a seperate message showing the patient as critically injured
+			else
+				failReason = SCR_EConsumableFailReason.ACE_MEDICAL_CRITICALLY_INJURED;
```

Comment:
```suggestion
				failReason = SCR_EConsumableFailReason.ACE_MEDICAL_TOO_DAMAGED_MEDICAL_FACILITY_REQUIRED;
```

Reply:
Missed updating this variable, it's been updated to the recommended name.